### PR TITLE
Feature/lp_1_Notification_UI

### PR DESCRIPTION
**Pull request by dmroldanch**
Notification UI

https://devaptivist.atlassian.net/browse/LP-1?atlOrigin=eyJpIjoiYzkyN2JhNjI1NjFhNGU3NmE2Zjk1Y2JhZGJhZTFlM2UiLCJwIjoiaiJ9

Create Notification UI 

NotificationScreen

### CODE REVIEW

File: feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.notification.R
+import com.iteneum.notification.domain.models.NotificationDomain
+
+@Composable
+fun NotificationListView() {
```
Comment (melissaquezada94) [inline feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt:20]: Please add the documentation to this function

File: feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                Text(
+                    text = stringResource(id = R.string.notification_today),
+                    modifier = Modifier.padding(top = dp16, bottom = dp16),
+                    color = MaterialTheme.colorScheme.tertiary,
+                    fontWeight = FontWeight.Medium,
+                    fontSize = MaterialTheme.typography.titleMedium.fontSize
+                )
+            }
+            items(todayNotificationList) { notification ->
+                Column(
```
Comment (melissaquezada94) [inline feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt]: If you are not going to add code inside the Column you could remove the  ( )

File: feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt (hunk 1 above)
Comment (dmroldanch) [reply feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt:20]: I added comments

File: feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt (hunk 2 above)
Comment (dmroldanch) [reply feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationListView.kt]: I removed the ( )

**Review by melissaquezada94 — APPROVED** [review]

**Review by carlosph816 — APPROVED** [review]
Well done!

**Pull request merged**